File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/Image.java
package org.metabrainz.mobile.data.sources.api.entities;

import java.util.ArrayList;

public class Image {
    private ArrayList<String> types= new ArrayList<>();
    private boolean front,back;
    private String image;

    public ArrayList<String> getTypes() {
        return types;
    }

    public void setTypes(ArrayList<String> types) {
        this.types = types;
    }

    public boolean isFront() {
        return front;
    }

    public void setFront(boolean front) {
        this.front = front;
    }

    public boolean isBack() {
        return back;
    }

    public void setBack(boolean back) {
        this.back = back;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }
}

File: app/src/main/java/org/metabrainz/mobile/data/sources/api/entities/Image.kt
package org.metabrainz.mobile.data.sources.api.entities

import com.google.gson.annotations.SerializedName
import org.metabrainz.mobile.data.sources.api.entities.Image.Thumbnail
import java.util.ArrayList

class Image {
    var types = ArrayList<String>()
    var isFront = false
    var isBack = false
    var image: String? = null
    var thumbnails: Thumbnail? = null

    class Thumbnail {
        var small: String? = null
        var large: String? = null
    }
}